pipeline-templates/sbom.yaml

```diff
@@ -0,0 +1,25 @@
+parameters:
+  pool: ''
+
+jobs:
+- job: SBOM
+  pool:
+    ${{ if eq(parameters.useOneEngineeringPool, 'true') }}:
+        ${{ parameters.pool }}
+    ${{ else }}:
+        vmImage: ${{ parameters.pool.vmImage }}
+  displayName: 📝 SBOM Generation
+  dependsOn:
+  - ${{ parameters.pool.os }}_Build
+  - ${{ parameters.pool.os }}_Package
+  steps:
+    - task: ManifestGeneratorTask@0
+      displayName: 📃 SBOM generation
+      inputs:
+        BuildDropPath: $(System.DefaultWorkingDirectory)
```

Comment:
nit: It is a stylistic choice, but this task item could be un-indented by 2 spaces (to make it the same alignment style as `dependsOn` and `outputs` arrays).

Reply:
Thanks for taking a look :) I think I like this approach since the step is still on the same indentation level, but I see your point